test/unit/clarity_ext/dilution/hamilton_tests.py

```diff
@@ -17,6 +20,9 @@ class HamiltonTests(unittest.TestCase):
     """
 
     def setUp(self):
+        self.mock = DilutionMock1()
```

Comment:
I have another branch that I would like to show you that shows another way of doing this. It doesn't require 100 lines of code ;)